clause/expression.go

```diff
@@ -130,7 +116,15 @@ func (expr NamedExpr) Build(builder Builder) {
 		} else if v == ' ' || v == ',' || v == ')' || v == '"' || v == '\'' || v == '`' || v == '\r' || v == '\n' || v == ';' {
 			if inName {
 				if nv, ok := namedMap[string(name)]; ok {
-					builder.AddVar(builder, nv)
+					if afterParenthesis {
+						if _, ok := nv.(driver.Valuer); ok {
+							builder.AddVar(builder, nv)
+						} else {
```

Comment:
those logic can add to the expandSliceOrArray method also?

Reply:
fixed